build/ci/official.yml

```diff
@@ -129,12 +129,29 @@ steps:
     ArtifactType: Container
   condition: succeededOrFailed()
 
-- task: NuGetPublisher@0
-  displayName: NuGet Publisher - Roslyn Myget
+- task: ms-vseng.MicroBuildShipTasks.7c429315-71ba-4cb3-94bb-f829c95f7915.MicroBuildCodesignVerify@2
+  displayName: Verify Signed Files
```

Comment:
Do we currently sign our packages? Have you tried this on a signed build?

Reply:
when i look at this package with "nuget verify -all microsoft.visualstudio.projectsystem.managed.16.7.0-beta1-65318-02.nupkg"
it looks like that package is author signed by microsoft.

Reply:
Yes, it is signed. This verification should be added and it's one of the requirements when pushing to vs-impl. When talking with James Williams he suggested this be done in the release pipeline instead though

Reply:
Opened https://github.com/dotnet/project-system/issues/6602 to track this